apps/jobbrowser/src/jobbrowser/tests.py

```diff
@@ -619,8 +623,15 @@ def setUp(self):
     api = MockImpalaQueryApi('http://url.com')
     self.api = QueryApi(None, impala_api=api)
 
+  def handle_query_start_time(self, start_time):
+    query_start_time = datetime.strptime(start_time[:-3], "%Y-%m-%d %H:%M:%S.%f"). \
+      replace(tzinfo=pytz.utc).astimezone(localtime._get_localzone()).strftime("%Y-%m-%d %H:%M:%S.%f")
+    return query_start_time
+
   def test_apps(self):
     response = self.api.apps({})
+    query_start_time = datetime.strptime('2017-10-25 15:38:26.637010000'[:-3], "%Y-%m-%d %H:%M:%S.%f").\
```

Comment:
Nit: To remove this because we are already using `handle_query_start_time` method.

Reply:
Nice catch, removed it.